src/html5sortable.ts

```diff
@@ -345,7 +345,13 @@ export default function sortable (sortableElements, options: configuration|objec
       originContainer = sortableContainer
 
       // add transparent clone or other ghost to cursor
-      setDragImage(e, dragItem, options.customDragImage)
+      let dragImage: Function | NodeList = null
+      if (typeof options.customDragImage === 'string') {
+        dragImage = document.querySelectorAll(options.customDragImage)
```

Comment:
Hey @yuki-katayama do you use `querySelector`**`All`** on purpose? Wouldn't it be better to use `querySelector` in case multiple items with e.g. a class exist?

Reply:
Hi @lukasoppermann 

Yes, I used querySelectorAll on purpose. Although querySelector is sufficient for the current implementation targeting a single element, I thought that using querySelectorAll would allow for future enhancements.

This method provides flexibility in handling scenarios where custom drag image functionality is required among multiple elements.
We thought this would ensure adaptability to different use cases and increase the scalability and maintainability of the implementation.

However, I think one is sufficient for this use case. Shall we fix this?

Reply:
Hi @lukasoppermann. Thank you for your review.

I think so too about using querySelector instead. At first, I chose querySelectorAll considering the future prospects of the implementation, but I thought it would be a good idea to implement it again when it becomes necessary. Also, when using querySelector, the code now defaults to the defaultDragImage function instead of throwing an error if the element is not found. Therefore, we changed the error handling from throwing an exception to logging the error using console.error. Please review the changes and let us know if further adjustments are needed.

Thank you for your feedback!

Reply:
test results.

 PASS  __tests__/setDragImage.test.ts
 PASS  __tests__/options.test.ts
 PASS  __tests__/sortableMethodsTests/_removeItemData.test.ts
 PASS  __tests__/serialize.test.ts (5.065s)
 PASS  __tests__/api.test.ts (5.2s)
 PASS  __tests__/isConnected.test.ts (5.241s)
 PASS  __tests__/events/events.test.ts (5.536s)
 PASS  __tests__/elementHeight.test.ts
 PASS  __tests__/sortableMethodsTests/_removeItemEvents.test.ts
 PASS  __tests__/filter.test.ts
 PASS  __tests__/elementWidth.test.ts
 PASS  __tests__/events/hoverClass.test.ts
 PASS  __tests__/store/storeConfig.test.ts
 PASS  __tests__/getHandles.test.ts
 PASS  __tests__/makePlaceholder.test.ts
 PASS  __tests__/insertAfter.test.ts
 PASS  __tests__/throttle.test.ts
 PASS  __tests__/store/storeData.test.ts
 PASS  __tests__/store/storePlaceholder.test.ts
 PASS  __tests__/index.test.ts
 PASS  __tests__/insertBefore.test.ts
 PASS  __tests__/offset.test.ts
 PASS  __tests__/isInDom.test.ts
 PASS  __tests__/debounce.test.ts
 PASS  __tests__/store/store.test.ts
-------------------------|----------|----------|----------|----------|-------------------|
File                     |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-------------------------|----------|----------|----------|----------|-------------------|
All files                |     86.1 |    74.62 |     89.9 |    86.19 |                   |
 attribute.ts            |      100 |      100 |      100 |      100 |                   |
 data.ts                 |      100 |      100 |      100 |      100 |                   |
 debounce.ts             |      100 |      100 |      100 |      100 |                   |
 defaultConfiguration.ts |      100 |      100 |      100 |      100 |                   |
 elementHeight.ts        |      100 |      100 |      100 |      100 |                   |
 elementWidth.ts         |      100 |      100 |      100 |      100 |                   |
 eventListener.ts        |      100 |      100 |      100 |      100 |                   |
 filter.ts               |      100 |      100 |      100 |      100 |                   |
 getEventTarget.ts       |      100 |    66.67 |      100 |      100 |                 6 |
 getHandles.ts           |      100 |       70 |      100 |      100 |             20,25 |
 getIndex.ts             |      100 |      100 |      100 |      100 |                   |
 hoverClass.ts           |      100 |      100 |      100 |      100 |                   |
 html5sortable.ts        |    75.55 |    49.03 |    76.74 |    76.19 |... 24,645,646,660 |
 insertHtmlElements.ts   |      100 |      100 |      100 |      100 |                   |
 isConnected.ts          |      100 |      100 |      100 |      100 |                   |
 isInDom.ts              |      100 |      100 |      100 |      100 |                   |
 makePlaceholder.ts      |      100 |      100 |      100 |      100 |                   |
 offset.ts               |      100 |      100 |      100 |      100 |                   |
 serialize.ts            |      100 |      100 |      100 |      100 |                   |
 setDragImage.ts         |      100 |      100 |      100 |      100 |                   |
 store.ts                |      100 |      100 |      100 |      100 |                   |
 throttle.ts             |      100 |      100 |      100 |      100 |                   |
-------------------------|----------|----------|----------|----------|-------------------|

Test Suites: 25 passed, 25 total
Tests:       7 skipped, 123 passed, 130 total
Snapshots:   0 total
Time:        10.198s
Ran all test suites.
$ standard 'src/*.ts' '__tests__/*.ts' | snazzy